src/components/Home/index.js

```diff
@@ -14,7 +14,7 @@ const Home = () => {
         >
           CodeBuddies
         </Box>
-        <Box component="h2" lineHeight={1.5} mb={4}>
+        <Box component="h3" lineHeight={1.5} mb={4} fontWeight={400}>
```

Comment:
This should be `h2` for accessibility and semantic reasons. If we need to change the font size, let's make a separate class for it instead.

Reply:
Hmm... I felt weird giving a non-headline paragraph of text an `h2` when it felt more like a `p` tag to me, and the title above it might have been an h2. The font weight solves the bolding issue at least, so will move it back to an h2 since you're right that the headings should be in order (e.g. h1, h2, h3).